ci-operator/config/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main__dast-418.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: dast/Dockerfile
+  to: ocp-qe-ci-dast
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: security-first-half
+  cron: 0 16 */21 * *
```

Comment:
Should this be changed?

Reply:
We want the latest version to be run every 3 weeks for our 4.next testing